### DISCUSSION TITLE
Modifying the footer

### DISCUSSION THREAD

**Discussion by AntonioSco** [templaza/astroid-framework#916] [Q&A]
Hi, 
I was watching a [video ](https://www.youtube.com/watch?v=IsCbRIfo570)explaining how to manage Astroid's footer, and I just realised that the customization interface for the template two (at least) doesn't have any footer section, to my big surprise. The interface being in that video was related to an older version of Astroid, 3.0. Does it mean that the latest Astroid has eliminated that customization section altogether? If so can somebody explain how to easily access the footer(s) so that can be customized?

**Comment by sonvnn** [comment]
You can found bottom sublayout in Sublayout tab.
Template options - Sublayouts

**Reply by AntonioSco** [reply]
Hi @sonvnn 
are you referring to this?
![imagen](https://github.com/user-attachments/assets/d2074407-f621-4950-ba81-b555bf938e27)
And I cannot see any "bottom sublayout tab" there
But there is nothing there related to footers...would you please explain?

**Comment by Banuk1616** [comment]
The video you have viewed is six years old and still from Astroid 1. Astroid has evolved since then. If you want to change the area shown below on your website, you can do this via the Layout Manager.

<img width="1181" alt="p1" src="https://github.com/user-attachments/assets/4554a3d4-e10e-4d1e-86d4-33b7114f3163" />

In a not quite up-to-date Template Two that I have installed on my development environment, this can be found under Footer section.

<img width="1068" alt="p2" src="https://github.com/user-attachments/assets/e1bcff53-09f1-400f-aeed-989ae2bff43d" />

You can also customize this area by creating your own sections, rows and columns. 

Best reagrds

**Reply by AntonioSco** [reply]
Oh, That's easier than I thought! Thanks mate!
